spark-tpcds-benchmark-runner/src/main/java/com/palantir/spark/tpcds/config/HadoopConfiguration.java

```diff
@@ -36,17 +41,47 @@
 
     Map<String, String> hadoopConf();
 
+    @Value.Default
+    default Map<String, FilesystemConfiguration> filesystems() {
+        return ImmutableMap.of("local", SimpleFilesystemConfiguration.of("file:///tmp/spark-benchmark"));
+    }
+
+    @Value.Default
+    default String defaultFilesystem() {
+        return "local";
```

Comment:
wondering if we need defaults for these in code - feel like it'll be too much magic for users if they don't specify any filesystems (and we ship with default config in the yml anyway)

Reply:
done